packages/x-components/src/x-modules/related-prompts/components/related-prompt.vue

```diff
@@ -64,39 +50,29 @@
         type: Object as PropType<RelatedPrompt>,
         required: true
       },
-      nextQueryButtonClass: {
-        type: String,
-        default: 'x-button-outlined'
```

Comment:
I think we should keep this level of customization, so we can add a class to modify spacing or add background colors as per customer's requirements.